docs/release-notes/9.0.asciidoc

```diff
@@ -4,29 +4,65 @@
 coming::[9.0.0]
 
 [discrete]
-[[release-notes-9.0.0]]
-=== 9.0.0
+[[release-notes-9.0.0-beta]]
+=== 9.0.0-beta
 
 [discrete]
-[[known-issue-9.0.0]]
+[[breaking-changes-9.0.0-beta]]
+==== Breaking changes
+* Removes deprecated bulk action API endpoints ({kibana-pull}207906[#207906]).
+* Refactors the Timeline HTTP API endpoints ({kibana-pull}200633[#200633]).
+* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
+* Disables deprecated API endpoints for bulk CRUD actions ({kibana-pull}197422[#197422]).
+
+[discrete]
+[[deprecations-9.0.0-beta]]
+==== Deprecations
+* Renames the `integration-assistant` plugin to `automatic-import` to match the associated feature ({kibana-pull}207325[#207325]).
+* Removes all legacy risk engine code and features ({kibana-pull}201810[#201810]).
+* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
+* Deprecates the SIEM signals migration APIs ({kibana-pull}202662[#202662]). 
+
+[discrete]
+[[known-issue-9.0.0-beta]]
 ==== Known issues
 
+// tag::known-issue[]
 [discrete]
-[[breaking-changes-9.0.0]]
-==== Breaking changes
+.Duplicate alerts can be produced from manually running threshold rules 
```

Comment:
Was this in 8.17 or this is the next release since these known issues were reported? I thought I read this in another release notes, but could be totally wrong.

Reply:
This was first introduced as a known issue in [8.16](https://www.elastic.co/guide/en/security/8.16/release-notes-header-8.16.0.html#release-notes-8.16.0), and we've been carrying it forward through the releases since it hasn't been fixed yet. 

Since there's no ETA on the fix, I've been considering moving this out of the known issues and into the feature docs. If that sounds like a plan, I'll remove this and the other manual rule runs known issue from the 9.0.0 beta release notes now and file a doc issue for myself to update the feature docs and release notes for 8.16-8.18 later.